tests/test-frameworks/mixed/test

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash -e
 . "$(dirname "$0")"/../../common.sh
 
-out_2_12=$(bazel test :mixed_2_12)
-echo "$out_2_12" | grep 'should pop values in last-in-first-out order'
-echo "$out_2_12" | grep 'JUnit many success'
-bazel test :mixed_2_11 | grep 'Dummy.bar: OK, proved property.'
-bazel test :mixed_2_11 --test_arg=--verbosity=HIGH | grep 'org.scalacheck.ScalaCheckFramework: 1 tests'
-bazel test :mixed_2_11 --test_arg=--verbosity=MEDIUM | grep -v 'org.scalacheck.ScalaCheckFramework: 1 tests'
+function test ()  {
+  out=$(bazel test ":mixed_$1")
+  echo "$out" | grep 'should pop values in last-in-first-out order'
```

Comment:
Is this checking for the line that was removed from the previous file? This should perhaps also be changed

Reply:
I upgrade ScalaTest as part of this PR and the old test no longer compiled with the new version. So I created a new dummy test.

Reply:
Good catch on that grep. Putting it inside of a bash function broke the exit on error, making this test pass every time. After re-reading the original test, it's also not a good idea to make this a bash function because the code isn't duplicated.

Everything should be fixed now and the test should once again be useful.